roles/identity-management/manage-rh-sso-identities/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Process all tasks
+  include_tasks: process_all_tasks.yml
+  loop: "{{ identities.users }}"
+  loop_control:
+    loop_var: user_data
+  when:
+    - identities.targets is undefined or 'rh-sso' in identities.targets
+  tags:
+    - create-user
```

Comment:
Since we'll always want to run this task, you can actually use a tag called `always`, that way you don't need to specify a tag for all steps you might want to include here.

Reply:
Awesome.